pyvo/registry/regtap.py

```diff
@@ -399,10 +400,22 @@ def __init__(self, access_url, *, standard_id=None,
         else:
             self.is_vosi = False
 
+        # a service is user visible if it has a corresponding service class
+        if self.standard_id is not None and self.standard_id != "":
+            service_type = self.standard_id.split("#")[0]  # remove possible suffixes
```

Comment:
The technical term for "suffix" would be "standards key", but I share your doubts whether that's an improvement in comprehensability.